### PR TITLE
added typing for ListenToHopReady

### PR DESCRIPTION
**Pull request by Jxl-s**
for commit https://github.com/MadStudioRoblox/ProfileService/commit/1f83c0cee410e426f9a4916490d65fd1bea9f8c4

### CODE REVIEW

File: profileservice/globals.d.ts [hunk 1]
```diff
@@ -204,6 +204,13 @@ export interface ViewProfile<DataType extends object> {
 	 */
 	Release(): void;
 
+	/**
+	 * In many cases ProfileService will be fast enough when loading and releasing profiles as the player teleports between places belonging to the same universe / game. However, if you're experiencing noticable delays when loading profiles after a universe teleport, you should try implementing `:ListenToHopReady()`.
+	 *
+	 * A listener passed to `:ListenToHopReady()` will be executed after the releasing UpdateAsync call finishes. `:ListenToHopReady()` will usually call the listener in around a second, but may ocassionally take up to 7 seconds when a profile is released next to an auto-update interval (regular usage scenario - rapid loading / releasing of the same profile key may yield different results).
+	 */
+	 ListenToHopReady(listener: () => void): RBXScriptSignal;
+
```
Comment (OverHash) [inline profileservice/globals.d.ts]: The formatting here is a bit wonky, can we try this?
```suggestion
	/**
	 * In many cases ProfileService will be fast enough when loading and releasing profiles as the player teleports between places belonging to the same universe / game. However, if you're experiencing noticeable delays when loading profiles after a universe teleport, you should try implementing `:ListenToHopReady()`.
	 *
	 * A listener passed to `:ListenToHopReady()` will be executed after the releasing UpdateAsync call finishes. `:ListenToHopReady()` will usually call the listener in around a second, but may occasionally take up to 7 seconds when a profile is released next to an auto-update interval (regular usage scenario - rapid loading / releasing of the same profile key may yield different results).
	 *
	 * In short, `Profile:ListenToRelease()` and `Profile:ListenToHopReady()` will both execute the listener function after release, but `Profile:ListenToHopReady()` will additionally wait until the session lock is removed from the `Profile`.
	 * @param listener The listener to execute after the releasing UpdateAsync call finishes.
	 */
```

**Review by OverHash — CHANGES_REQUESTED** [review]
Just this small formatting issue, and this looks good!

Thanks for your contribution!

**Review by OverHash — APPROVED** [review]
